ai/model-serving-tensorflow/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: tf-serving-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  rules:
+    - host: tf.example.com
```

Comment:
Do we need the `host` key here? Can we remove it or make it optional so that DNS isn't required for this example? If removing / making this optional, change `<ingress-host>` in README to `<ingress-host-or-ip>` as well.